src/math/Interpolate.cc

```diff
@@ -161,6 +161,9 @@ ::gsl_interp_type const *styleToGslInterpType(Interpolate::Style const style) {
             throw LSST_EXCEPT(pex::exceptions::LogicError,
                               str(boost::format("You can't get here: style == %") % style));
     }
+    // don't use default statement to let compiler check switch coverage
+    throw LSST_EXCEPT(pex::exceptions::LogicError,
+                      str(boost::format("You can't get here: style == %") % style));
```

Comment:
An abort would cause the entire Stack to crash, with no explanation, in the middle of processing. There is no justification for calling `abort()` in low-level functions in modern code.